src/test/scala/chiselTests/DisableSpec.scala

```diff
@@ -3,7 +3,6 @@
 package chiselTests
 
 import chisel3._
-import chisel3.testers.BasicTester
 import _root_.circt.stage.ChiselStage
```

Comment:
Nit: unrelated change. Feel free to land separately and directly.

![6mkijk](https://github.com/chipsalliance/chisel/assets/1018530/2bd0dd3f-4602-4370-bb92-78d4fa261bf4)

Reply:
Lol removed.